docs/api/index.md

```diff
@@ -149,13 +149,15 @@ You cannot use this syntax, when using Vitest as [type checker](/guide/testing-t
   test.todo.concurrent(/* ... */) // or test.concurrent.todo(/* ... */)
   ```
 
-  When using Snapshots with async concurrent tests, due to the limitation of JavaScript, you need to use the `expect` from the [Test Context](/guide/test-context.md) to ensure the right test is being detected.
+  When using Snapshots with async concurrent tests, you need to use `expect` from the [Test Context](/guide/test-context.md) and call `expect.assertions` to ensure the test will detect any failures.
```

Comment:
No, I did not mean that. You don't need to call `expect.assertions` in this case. You need to use local expect, **if** you are calling `assertions` in a `concurrent` test. This is also true for your other edits. We just need to mention that this API depends on the local expect in this case.